internal/attestation/renderer/renderer.go

```diff
@@ -172,3 +184,105 @@ func getPassFromTerm(confirm bool) ([]byte, error) {
 	}
 	return pw1, nil
 }
+
+func (ab *AttestationRenderer) getSigner(ctx context.Context) (sigstoresigner.Signer, error) {
+	var (
+		signer sigstoresigner.Signer
+		err    error
+	)
+
+	if ab.signingKeyPath != "" {
+		ab.logger.Debug().Str("path", ab.signingKeyPath).Msg("loading key")
+		signer, err = signature.SignerFromKeyRef(context.Background(), ab.signingKeyPath, getPass)
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		// key is not provided, let's create one
+		ab.logger.Info().Msg("key not provided, running in key-less mode")
```

Comment:
debug too?

Reply:
I was thinking this message would be useful in normal mode.